source/GameInterface/Services/Clans/Messages/ClanNameChange.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Common.Messaging;
+using ProtoBuf;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using TaleWorlds.CampaignSystem;
+
+namespace GameInterface.Services.Clans.Messages
+{
+    public record ClanNameChange : IEvent
```

Comment:
Please add summary comments
```suggestion
    public record ClanNameChange : ICommand
```